conf/nginx/templates/nginx.conf.web.http.default.template

```diff
@@ -50,6 +50,17 @@ server
         proxy_http_version 1.1;
     }
 
+    location /services/chats/events
```

Comment:
Is the absence of a following slash intended and verified to ensure that we do not receive an unnecessary redirect in response to the request recieved at this endpoint?

ref: [from nginx docs](http://nginx.org/en/docs/http/ngx_http_core_module.html#location), 
>If a location is defined by a prefix string that ends with the slash character, and requests are processed by one of [proxy_pass](http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_pass), [fastcgi_pass](http://nginx.org/en/docs/http/ngx_http_fastcgi_module.html#fastcgi_pass), [uwsgi_pass](http://nginx.org/en/docs/http/ngx_http_uwsgi_module.html#uwsgi_pass), [scgi_pass](http://nginx.org/en/docs/http/ngx_http_scgi_module.html#scgi_pass), [memcached_pass](http://nginx.org/en/docs/http/ngx_http_memcached_module.html#memcached_pass), or [grpc_pass](http://nginx.org/en/docs/http/ngx_http_grpc_module.html#grpc_pass), then the special processing is performed. In response to a request with URI equal to this string, but without the trailing slash, a permanent redirect with the code 301 will be returned to the requested URI with the slash appended

Reply:
This endpoint is related to a websocket and we don't want anything after "events".